### PR TITLE
docs: Added bug report issue template for Next.js Closes #370

### LINKED ISSUE #370: Create a Bug Report Issue Template for Next.js
## Feature Request

Is your feature request related to an existing [Issue](https://www.github.com/Auto-DL/Generator/issues) or a new implementation?

No

### Feature Description

An Issue Template specifically catered towards the newer V2 Frontend developed in Next.js.
The template must be for **reporting a new bug** in the previously existing codebase (`FrontEndApp/autodl-platform`).

### Possible Solution

Create a new `NextJS Bug.md` file in `.github/ISSUE_TEMPLATE` directory.

### Additional Context

Remember to add comments in the template so as to guide the user appropriately (when the template is used).

### PR DESCRIPTION
**Pull request by Aniket762**
# Pull Request

## What does this PR do?
Fixes #370

## What part does this affect?
- [ ] FrontEnd.
- [ ] BackEnd.
- [X] Documentation.
- [ ] Other. (Please specify below)

## Before submitting
- [X] Was this discussed/approved via a GitHub issue or slack?
- [X] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [X] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [X] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [X] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure the code is clean and docstrings have been added or updated as required?
- [ ] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [X] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

**Comment by Aniket762** [comment]
Hey, I have sent the Pull request as assigned. If some changes are needed let me know would be more than happy to resolve it. I have not added any emojis in the bug report template let me know if you want me to add them. 

I was thinking of adding the line
` Please do star the repo to increase visibility and help us reach a bigger audience because this project is ours as much as yours `

Let me know your thoughts on this! 

 Once this PR gets reviewed and closed I will start working on #369

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [inline .github/ISSUE_TEMPLATE/NextJS Bug.md:8]: Remove the table. All of this is already on github we don't need a separate table for it.

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [inline .github/ISSUE_TEMPLATE/NextJS Bug.md:21]: Remove this too.

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [inline .github/ISSUE_TEMPLATE/NextJS Bug.md:37]: comment this out

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [inline .github/ISSUE_TEMPLATE/NextJS Bug.md:59]: comment this out.

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [inline .github/ISSUE_TEMPLATE/NextJS Bug.md:63]: remove the backticks, and solve the grammar issues.
Instead of  `challenge code` try ` `erroneous code`.

**Review by bamblebam — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (ADI10HERO) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md:8]: As it's a Nextjs issue, can you also add the label "Next" to the labels? I am pretty sure we can add multiple labels...

**Review by ADI10HERO — COMMENTED** [review]
I did not peruse the whole yaml, but as it's a frontend bug report, maybe we can have an optional screenshot section (please ignore if already added)

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (RusherRG) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md:59]: Instead of commenting the entire thing out... we can just comment out the lines 49 to 58 which could serve as an example rather and 46-47 being compulsory fields. Just for better understanding

Comment (bamblebam) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md:59]: That could be done but don't keep the node version compulsory. The user might be just browsing and might not know what node version he is using.

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (Aniket762) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md:8]: Okay, I am adding the label Next to it, and how do I not remove the table and automate it. As per my knowledge, if I comment it out, automation will stop. Can you please guide me!

**Comment by Aniket762** [comment]
> I did not peruse the whole yaml, but as it's a frontend bug report, maybe we can have an optional screenshot section (please ignore if already added)

Hey, @ADI10HERO I have commented that under `Additional context/Screenshots`. So it's present in the report template.

**Comment by Aniket762** [comment]
I have made the changes please review it and let me know if I need to change something else.

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (bamblebam) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md]: don't remove the table

Comment (Aniket762) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md]: I have sent the PR and haven't removed it. Can you please review it?

**Review by McTechie — CHANGES_REQUESTED** [review]
Just a few minor changes and it's good to go! xD

File: .github/ISSUE_TEMPLATE/NextJS Bug.md
Comment (Aniket762) [reply .github/ISSUE_TEMPLATE/NextJS Bug.md:69]: Okay removing rn

**Comment by Aniket762** [comment]
@McTechie I have made the changes please review it.

**Review by McTechie — APPROVED** [review]
Looks good to me! xD

**Comment by McTechie** [comment]
Finally before merging this PR you could edit the `Docs` in the title to `docs` to make sure all the workflows pass! 😄

**Comment by Aniket762** [comment]
Hey, @RusherRG @McTechie are there any changes I need to do? Else I guess we are good to go. Once this gets merged I will start working on the second issue assigned to me.

**Comment by RusherRG** [comment]
> Hey, @RusherRG @McTechie are there any changes I need to do? Else I guess we are good to go. Once this gets merged I will start working on the second issue assigned to me.

You can start working on the next issue. We needed to discuss a few things among us related to the templates and hence haven't merged/accepted the changes yet.

**Comment by Aniket762** [comment]
> > Hey, @RusherRG @McTechie are there any changes I need to do? Else I guess we are good to go. Once this gets merged I will start working on the second issue assigned to me.
> 
> You can start working on the next issue. We needed to discuss a few things among us related to the templates and hence haven't merged/accepted the changes yet.

Actually, my second issue #370  was also related to the issue template generation so I was waiting for this to get merged so that I can understand the way I need to document it. It's okay I will wait till then work on some other repos 🙈

**Comment by ADI10HERO** [comment]
Good work!

**Pull request closed without merging**
